modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheValidateIndexes.java

```diff
@@ -186,20 +189,20 @@ public UUID nodeId() {
                     ValidateIndexesPartitionResult res = e.getValue();
 
                     if (!res.issues().isEmpty()) {
-                        logger.logWithIndent(CommandLogger.join(" ", "SQL Index", e.getKey(), e.getValue()));
+                        logger.info(INDENT + CommandLogger.join(" ", "SQL Index", e.getKey(), e.getValue()));
 
                         for (IndexValidationIssue is : res.issues())
-                            logger.logWithIndent(is, 2);
+                            logger.info(DOUBLE_INDENT + is);
                     }
                 }
             }
 
             if (!errors)
-                logger.log("no issues found.");
+                logger.severe("no issues found.");
```

Comment:
seems that should be 'log' instead of 'severe'

Reply:
fixed